html/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../styles/main.css">
+    <title>WebStore</title>
+</head>
+<body>
+<div class="page-wrapper">
+    <header id="header">
+        <div>
+            <a href="index.html" id="logo"><img src="../img/logo/logo.png" alt="logo"></a>
+        </div>
+        <div id="menuToggle">
+            МЕНЮ
```

Comment:
I see that you based on english, but there is russian here. This confuses. It is unexpected for client. Please, you should base on single language.